app/src/main/java/org/oppia/app/player/exploration/ExplorationActivity.kt

```diff
@@ -0,0 +1,16 @@
+package org.oppia.app.player.exploration
+
+import android.os.Bundle
+import org.oppia.app.activity.InjectableAppCompatActivity
+import javax.inject.Inject
+
+/** The starting point for exploration*/
+class ExplorationActivity : InjectableAppCompatActivity() {
+  @Inject lateinit var explorationActivityController: ExplorationActivityController
```

Comment:
I started calling these presenters in #106. Let's rename them to be presenters in this PR, too, to avoid the ambiguity of 'controller' here.

Reply:
Changed Fragment and Activity `Controller`s to `Presenter`s